src/brevitas_examples/llm/main.py

```diff
@@ -285,6 +288,17 @@ def main(args):
 
     model = offload_model(model)
 
+    if args.learned_round:
+        print("Applying learned round...")
+
+        learned_round_llm_utils = LearnedRoundLLMUtils()
```

Comment:
Ideally, we shouldn't be needing this class, and try to merge everything with the vision one.
For anything that can't be merged, it is user defined.

Reply:
My idea with this design is to decouple the logic of the PTQ algorithm, which is contained in LearnedRoundOptimizer, from anything that it is model/dataloader specific, e.g. capturing inputs/outputs for a block. This needs to be managed by specific utilities, which adhere to the interface LearnedRoundModelUtils (thus following the strategy pattern). By doing so, a potential change in the vision models would not break the LLM entrypoint, and viceversa.